pkg/application/cluster_app.go

```diff
@@ -24,13 +24,21 @@ type Cluster struct {
 	ClusterApp     *Application
 	DefaultAppsApp *Application
 	Organization   *organization.Org
+	Release        ReleasePair
 }
 
 type AppPair struct {
 	App       *applicationv1alpha1.App
 	ConfigMap *corev1.ConfigMap
 }
 
+type ReleasePair struct {
+	Version string
```

Comment:
is `Version` the same as the `name` of the `Release`? I was wondering if `name` would be a better name for this field.

Reply:
The SDK uses "version" - https://github.com/giantswarm/releases/blob/a703ee1d9819ee9762568ef00af42947afe6add6/sdk/api/v1alpha1/release.go#L129

Reply:
Ok then